cmd/snap-bootstrap/cmd_recovery_chooser_trigger.go

```diff
@@ -88,11 +88,16 @@ func (c *cmdRecoveryChooserTrigger) Execute(args []string) error {
 
 	err = triggerwatchWait(timeout)
 	if err != nil {
-		if err == triggerwatch.ErrTriggerNotDetected {
+		switch err {
+		case triggerwatch.ErrTriggerNotDetected:
 			logger.Noticef("trigger not detected")
 			return nil
+		case triggerwatch.ErrNoMatchingInputDevices:
+			logger.Noticef("no matching input devices")
```

Comment:
Perhaps you could log err string (also above) instead of explicit message?

Reply:
The logger is already set up to log to stderr or whichever other place that is correct in this context.

Reply:
Sorry, maybe I wasn't clear, what I meant is:
```
logger.Noticef("%s", err)
```